examples/tensorflow/image-classifier-inception/cortex_batch_sized.yaml

```diff
@@ -0,0 +1,16 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
```

Comment:
This comes without a README. The thinking is that there's already an example with the `resnet50` model. I only added the `cortex.yaml` config because the model already had that extra dimension for batches, and so I thought, why not (as it's a low-hanging fruit)?

What do you say?

Reply:
sounds good!